README.md

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+triangles() {
```

Comment:
This is good but I need to call triagles from my terminal every time, make it more interactive, I don't want the execution of the program to end unless I said otherwise (e.g. if I enter 0 => nothing else to classify (same if I enter stop))